cfbs/main.py

```diff
@@ -119,6 +119,13 @@ def main() -> int:
         return commands.update_command(args.args)
     if args.command == "input":
         return commands.input_command(args.args)
+    if args.command in ("set-input", "get-input"):
+        if len(args.args) < 1:
+            user_error("Missing argument <module-name>")
+        module, file = args.args[0], (args.args[1] if len(args.args) >= 2 else None)
+        if args.command == "set-input":
+            return commands.set_input_command(module, file)
+        return commands.get_input_command(module, file)
```

Comment:
As @vpodzime pointed out, you can consider adding a requirement to specify filename or `-` for stdin here;

```
cfbs get-input <module-name> <filename or - for stdin>
cfbs set-input <module-name> <filename or - for stdin>
```

Then you can always require `len(args.args) == 2`, and print out a helpful error message:

```
error: Filename required as last argument (or - for stdin)
```

I don't think we should make it more complicated than this by using `--`-options or prompts.